tests/unit/utils/test_materializer_utils.py

```diff
@@ -0,0 +1,124 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+import os
+import shutil
+import pytest
+import tempfile
+
+from pytest_mock import mocker
+from zenml.constants import MODEL_METADATA_YAML_FILE_NAME
+
+from zenml.utils.materializer_utils import (
+    METADATA_DATATYPE, 
+    METADATA_MATERIALIZER,
+    load_artifact, 
+    load_model_from_metadata, 
+    save_model_metadata,
+    _load_artifact,
+)
+from zenml.models import ArtifactResponseModel
+
+@pytest.fixture
+def model_artifact(mocker):
+    return mocker.Mock(spec=ArtifactResponseModel,
```

Comment:
what does returning a Mock instead of the actual artifact do here?

Reply:
I apologize if I did not fully understand the question. Is returning a mock object representing an `ArtifactResponseModel` an appropriate way to implement this?